package.json

```diff
@@ -17,7 +17,7 @@
     "build": "yarn build:info && next build",
     "build:info": "node .build-info.generate.js",
     "start": "next start",
-    "pretty": "prettier -w .",
+    "pretty": "prettier -w . --ignore-path=.prettierignore",
```

Comment:
If anyone is going through this PR and wondering why I added this diff, it is because Prettier 3.0.0 will use `.gitignore` by default from now on, and not `.prettierignore`. At the moment, I don't want to spend too much time adapting prettierignore vs gitignore